library/core/src/iter/traits/iterator.rs

```diff
@@ -1543,11 +1543,11 @@ pub trait Iterator {
     /// collection into another. You take a collection, call [`iter`] on it,
     /// do a bunch of transformations, and then `collect()` at the end.
     ///
-    /// One of the keys to `collect()`'s power is that many things you might
-    /// not think of as 'collections' actually are. For example, a [`String`]
-    /// is a collection of [`char`]s. And a collection of
-    /// [`Result<T, E>`][`Result`] can be thought of as single
-    /// [`Result`]`<Collection<T>, E>`. See the examples below for more.
+    /// `collect()` is powerful and can also crate instances of types that
```

Comment:
```suggestion
    /// `collect()` is powerful and can also create instances of types that
```
Is it really creating? I thought it is still reusing the existing data in some cases? Shouldn't it be convert type?

Reply:
I also don't think we need to repeat saying that it is powerful since it is mentioned in the above paragraph.